go/start/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"breakfast/app"
+	"context"
+	"log"
+
+	"go.temporal.io/sdk/client"
+)
+
+func main() {
+
+	// Create the client object just once per process
+	c, err := client.Dial(client.Options{})
+	if err != nil {
+		log.Fatalln("unable to create Temporal client", err)
+	}
+	defer c.Close()
+
+	options := client.StartWorkflowOptions{
+		ID:        "trip-planning-workflow",
+		TaskQueue: app.TripPlanningTaskQueue,
+	}
+
+	expiration := app.CreditCardExpiration{Year: 2023, Month: 5}
+	ccInfo := app.CreditCardInfo{Number: 123456789, Ccv: 123, Expiration: expiration}
+	info := app.BookingInfo{Name: "Emily", Address: "123 Temporal Lane", CcInfo: ccInfo}
```

Comment:
We also need to generate the `idempotencyKey` either here or in a side effect of the workflow

Reply:
Chad taught me we often just use the workflow run id, so I switched to that.

Reply:
yes sorry I missed that. Fixed.